.github/workflows/build_flatpak.yml

```diff
@@ -0,0 +1,90 @@
+# https://docs.github.com/en/actions/learn-github-actions/contexts
+
+---
+name: Tuxemon
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+  page_build:
+  workflow_dispatch:
+
+
+jobs:
+  flatpak:
+    name: "Flatpak"
+    runs-on: ubuntu-latest
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:kde-5.15-21.08
+      options: --privileged
+    steps:
+    - name: Show environment variables
+      run: >
+        echo IS_RELEASE: ${{ env.IS_RELEASE }}
+
+    # https://stackoverflow.com/questions/60916931/github-action-does-the-if-have-an-else
+    - name: Determine flatpak release branch
+      uses: haya14busa/action-cond@v1
+      id: flatpak_release_branch
+      with:
+        cond: ${{ env.IS_RELEASE }}
+        if_true: 'stable'
+        if_false: 'development'
+
+
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Clone flatpak manifest from Flathub
+      uses: GuillaumeFalourd/clone-github-repo-action@v1
+      with:
+        owner: 'flathub'
+        repository: 'org.tuxemon.Tuxemon'
+
+    - name: Checkout development branch (TODO: remove this once a new version of tuxemon is released)
```

Comment:
It doesn't like the colon after "TODO", change it to something else, e.g. "TODO - "